Sources/DistributedActors/Cluster/ClusterShell+LeaderActions.swift

```diff
@@ -99,7 +99,14 @@ extension ClusterShell {
         }
 
         system.cluster.updateMembershipSnapshot(state.membership)
-        eventsToPublish.forEach { state.events.publish($0) }
+
+        let eventStream = state.events
+        let events = eventsToPublish
+        Task {
+            for event in events {
+                await eventStream.publish(event)
```

Comment:
maybe just inline:

```suggestion
                await state.events.publish(event)
```

Reply:
or is it because sendable warnings?

Reply:
Gotcha, I see -- maybe we can `Task { [eventsToPublish] ` at least then?

Reply:
Can do this though `Task { [eventsToPublish, state] in`

Reply:
amended https://github.com/apple/swift-distributed-actors/pull/1012/commits/f80caa21f4a8a3d54b9c5926b3bd3df0a4ca2b76